src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArr = sourceString.trim().split(';');
+  const styleObject = {};
+
+  stylesArr.forEach((property) => {
+    const [key, value] = property.split(':').map(item => item.trim());
```

Comment:
<img width="1043" alt="image" src="https://github.com/mate-academy/js_style-to-object/assets/86789161/473540e0-152f-4e6b-99a6-2280e239a07a">